kubernetes-manifests/cartservice.yaml

```diff
@@ -38,8 +38,6 @@ spec:
         env:
         - name: REDIS_ADDR
           value: "redis-cart:6379"
-        - name: ASPNETCORE_URLS
-          value: "http://0.0.0.0:7070"
         resources:
```

Comment:
Did you test it deployed on kubernetes?

Reply:
Yes I did.

Reply:
I prefer to set environment variables for kubernetes clusters in kubernetes manifest instead of docker file. If you then decide to use buildbacks for container building, you might introduce errors in configuration as a result of handling environment-related stuff in dockerfile instead of kubernetes yaml

Reply:
Having it in the `Dockerfile` allows to have centralized way to get thei `cartservice` app working on any circunstances (`Docker`, Docker-compose`, `Kubernetes`, etc.). It's not a Kubernetes thing, for example today this command `docker run cartservice` won't work unless `-e ASPNETCORE_URLS="http://0.0.0.0:7070"` is passed.

Reply:
Done.